utilities/common.py

```diff
@@ -210,6 +210,7 @@ def __call__(self, parser, namespace, values, option_string=None):
         parser.add_argument("--sf2DnoUt", action='store_true', help="Use older smooth 2D scale factors with no ut dependence")
         parser.add_argument("--isoEfficiencySmoothing", action='store_true', help="If isolation SF was derived from smooth efficiencies instead of direct smoothing") 
         parser.add_argument("--noScaleFactors", action="store_true", help="Don't use scale factors for efficiency (legacy option for tests)")
+        parser.add_argument("--dataYear", default=2016, type=int, help="year for which the analysis is performed")
```

Comment:
instead of adding this option, can we make use of "--era" instead and add the choices "2017" and "2018"? 
You would replace instances of ```if args.dataYear == 2016``` with ```if args.era in ["2016preVFP", "2016postVFP"]```